policyengine_us/parameters/gov/states/me/tax/income/deductions/personal_exemption/phaseout/width.yaml

```diff
@@ -18,4 +21,4 @@ HEAD_OF_HOUSEHOLD:
 JOINT:
   2021-01-01: 125_000
 SURVIVING_SPOUSE:
-  2021-01-01: 62_500
```

Comment:
Was this change intended?

Reply:
The numerator of the fraction is the taxpayer's Maine adjusted gross income less the applicable amount, except that the numerator may not be less than zero, and the denominator is $62,500 in the case of a married individual filing a separate return and $125,000 in all other cases.[§5126-A](https://legislature.maine.gov/statutes/36/title36sec5126-A.html)